lib/Test/Nginx/Util.pm

```diff
@@ -636,6 +640,38 @@ sub error_log_data () {
     return \@lines;
 }
 
+sub check_prevblock_error_log($) {
+    my ($is_end) = @_;
+
+    my $block = $PrevBlock;
+
+    if (!$PrevBlock) {
+        return;
+    }
+
+    my $name = $block->name;
+
+    if (defined $block->shutdown_error_log) {
+        if (!$is_end && $UseHup) {
```

Comment:
I don't think `is_end` is needed here. We don't need to enable the tests for the last test block specifically under the HUP reload testing mode. It does not make much sense when the test blocks are shuffled by default and the running order is usually random.